src/cli/repl/core.ts

```diff
@@ -8,12 +8,14 @@ import { bold } from '../../statistics'
 import { prompt } from './prompt'
 import type { ReplOutput } from './commands'
 import { commandNames, getCommand, standardReplOutput } from './commands'
+
```

Comment:
space me :sparkles:

Reply:
i actually think that one weird eslint plugin rule thing we added a while back to combine imports occasionally puts empty lines for seemingly no reason ✨ truly a technological wonder